src/Paket.Core/PublicAPI.fs

```diff
@@ -192,6 +192,9 @@ type Dependencies(dependenciesFileName: string) =
             semVerUpdateMode = defaultArg semVerUpdateMode SemVerUpdateMode.NoRestriction,
             touchAffectedRefs = defaultArg touchAffectedRefs false)
 
+    static member GenerateIncludeScripts() =
```

Comment:
Going to rename that one, harmonizing on "load scripts" which better fits F# terminology